go/Dockerfile.expose-port

```diff
@@ -0,0 +1,11 @@
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS mirror
+
+RUN apk add --no-cache go musl-dev build-base
+
+ADD . /go/src/github.com/moby/vpnkit/go
+WORKDIR /go/src/github.com/moby/vpnkit/go
+
+RUN GOPATH=/go make build/vpnkit-forwarder.linux build/vpnkit-expose-port.linux
+
+FROM scratch
+COPY --from=mirror /go/src/github.com/moby/vpnkit/go/build/vpnkit-expose-port.linux /vpnkit-expose-port
```

Comment:
Since we're going to use `vpnkit-expose-port` as an init-container in linuxkit, it's probably better to put it in a location that makes sense when extracted in the host fs. E.g. `/usr/local/bin/vpnkit-expose-port`.